jina/optimizers/__init__.py

```diff
@@ -155,6 +155,13 @@ def optimize_flow(
         return result_processor(study)
 
 
+def run_optimizer(args):
+    from .flow_runner import SingleFlowRunner
+    with open(args.uses) as f:
+        optimizer = JAML.load(f)
+    result = optimizer.optimize_flow()
```

Comment:
is the result a OptimizerResultProcessor???

Then in the cli we should call the print method?

Reply:
The results are getting logged already.
Therefore, I think it would be redundant to print them again.
```
[I 2021-01-26 08:59:38,188] A new study created in memory with name: no-name-19a1acf1-c8d6-4493-95b2-1cce2c3961ea
[I 2021-01-26 08:59:39,942] Trial 0 finished with value: 0.0 and parameters: {'JINA_DUMMYCRAFTER_PARAM1': 0, 'JINA_DUMMYCRAFTER_PARAM2': 1, 'JINA_DUMMYCRAFTER_PARAM3': 2}. Best is trial 0 with value: 0.0.
[I 2021-01-26 08:59:41,678] Trial 1 finished with value: 1.0 and parameters: {'JINA_DUMMYCRAFTER_PARAM1': 0, 'JINA_DUMMYCRAFTER_PARAM2': 1, 'JINA_DUMMYCRAFTER_PARAM3': 1}. Best is trial 1 with value: 1.0.
[I 2021-01-26 08:59:43,481] Trial 2 finished with value: 0.0 and parameters: {'JINA_DUMMYCRAFTER_PARAM1': 0, 'JINA_DUMMYCRAFTER_PARAM2': 1, 'JINA_DUMMYCRAFTER_PARAM3': 2}. Best is trial 1 with value: 1.0.
[I 2021-01-26 08:59:45,229] Trial 3 finished with value: 1.0 and parameters: {'JINA_DUMMYCRAFTER_PARAM1': 0, 'JINA_DUMMYCRAFTER_PARAM2': 1, 'JINA_DUMMYCRAFTER_PARAM3': 1}. Best is trial 1 with value: 1.0.
[I 2021-01-26 08:59:46,984] Trial 4 finished with value: 0.0 and parameters: {'JINA_DUMMYCRAFTER_PARAM1': 0, 'JINA_DUMMYCRAFTER_PARAM2': 1, 'JINA_DUMMYCRAFTER_PARAM3': 2}. Best is trial 1 with value: 1.0.
```
Also you can specify the `output_file` in the yaml config to specify the path where the results should be stored.